include/deal.II/differentiation/sd/symengine_number_visitor_internal.h

```diff
@@ -20,18 +20,13 @@
 
 #ifdef DEAL_II_WITH_SYMENGINE
 
-DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
 // Low level
 #  include <symengine/basic.h>
 #  include <symengine/dict.h>
 #  include <symengine/symengine_exception.h>
 #  include <symengine/symengine_rcp.h>
 
 // Visitor
```

Comment:
this comment is now out of place. Maybe delete?

Reply:
Yes.